example/e2e/helpers.ts

```diff
@@ -259,6 +254,12 @@ export async function getBillingInfoAddress(): Promise<string> {
   } else return Selectors.ADDRESS_ONE_ENTRY_FIELD;
 }
 
+export async function getBillingInfoState(): Promise<string> {
+  if (await isIOS()) {
+    return Selectors.STATE_FIELD;
+  } else return Selectors.STATE_ENTRY_FIELD;
```

Comment:
Let's be consistent regarding blocks and use 
```

if (condition) {

} else {

}
```

Or, in this case, it can probably be:

```
export async function getBillingInfoState(): Promise<string> {
  if (await isIOS()) {
    return Selectors.STATE_FIELD;
  } 
  
  return Selectors.STATE_ENTRY_FIELD;
}
```